app/views/dashboard/show.html.erb

```diff
@@ -19,3 +19,19 @@
     <%= render Elements::ButtonLinkComponent.new(link: new_work_path(collection_druid: collection.druid), label: 'Deposit to this collection', variant: :primary) %>
   </div>
 <% end %>
+
+<% current_user.manages_collections.each do |collection| %>
```

Comment:
In H2, the current interface includes managed and deposit into collections on the dashboard with no distinction between them and the current users owned collections. This is simply copying that format to our current dashboard until any design refinement is complete.